src/pyTOPP/src/pyopenms/pytopp/ctdsupport.py

```
@@ -0,0 +1,813 @@
+from __future__ import annotations
+
+import os
+import shutil
+import sys
+import tempfile
+import textwrap
+from collections.abc import Mapping
+from dataclasses import dataclass, field
+from difflib import get_close_matches
+from typing import Iterable, Mapping, Sequence
```

Ruff (0.14.1)

Redefinition of unused `Mapping` from line 8

(F811)